### PR TITLE
Add the main engine

### PR DESCRIPTION
**Pull request by pbrus**
I've created a new Python script which can be called from the command line or import as module. This will be handy in the future. I specially left some place that you could write docstrings and create body functions. I've also added a simple readme file.

### CODE REVIEW

File: char_counter.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+try:
+    from argparse import ArgumentParser
+    from codecs import open as open_coding
+    from string import lowercase as ALPHABET
```
Comment (Jakub21) [inline char_counter.py]: In python 3.6 a ImportError is raised.

File: char_counter.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    a
+    b
+    ...
+    ü
+    ß
+    """
+    alphabet = basic_alphabet
+
+    for dl in diacritics_letters:
+        alphabet += dl
```
Comment (Jakub21) [inline char_counter.py]: Both diacritics_letters and basic_alpabet are of string type. Using:
 alphabet = basic_alphabet + diacritics_letters
 will give the same result and looks easier.

**Review by Jakub21 — CHANGES_REQUESTED** [review]
After implementing those minor changes or addressing my feedback the pull request will be merged to master.

File: char_counter.py (hunk 1 above)
Comment (pbrus) [reply char_counter.py]: The reason for that is I wrote the code in Python2. To achieve compatibility with Python3 we must import `ascii_lowercase`.

File: char_counter.py (hunk 2 above)
Comment (pbrus) [reply char_counter.py]: Yeah, at the beginning I converted `alphabet` to the list but your idea is really simpler and the code looks more intuitive.

File: char_counter.py (hunk 1 above)
Comment (Jakub21) [reply char_counter.py]: The command `from string import ascii_lowercase` works in Python3. You can use it.

**Review by Jakub21 — APPROVED** [review]

File: char_counter.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    Return
+    ------
+    A string with all letters.
+
+    Examples
+    --------
+    >>> from char_counter import supplement_alphabet
+    >>> german_alphabet = supplement_alphabet(u"äöüß")
+    >>> for ga in german_alphabet:
+    ...     print ga
```
Comment (Jakub21) [inline char_counter.py]: `print ga` does not work in Python3.6. Please replace it with `print(ga)`. The change will not make this example incompatible with Python2.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Characters counter
+Program parses a text file and performs some statistical values.
```
Comment (Jakub21) [inline README.md]: I think this will be better:

> Program parses a text file and performs some statistical analysis.

**Review by Jakub21 — COMMENTED** [review]
Just some changes in docstring and readme.

**Pull request merged**